scm-webapp/src/main/java/sonia/scm/plugin/PluginCenter.java

```diff
@@ -65,6 +66,12 @@ public void handle(PluginCenterAuthenticationEvent event) {
     pluginCenterResultCache.clear();
   }
 
+  @Subscribe
+  public void handle(ScmConfigurationChangedEvent event) {
+    LOG.debug("clear plugin center cache, because of {}", event);
+    pluginCenterResultCache.clear();
```

Comment:
Are we sure that we want to invalidate it on every config change? It may not be expensive to refetch this data, but.... If they are instances which are run by scripts (like our java e2e tests?) or even the scm dogus which are configured on each startup, this could increase the traffic for our plugin center?